FreeRTOS/FreeRTOS
# Delete HTTP demo task upon completion

Description
-----------

This causes problems in ports like Microchip where `prvTaskExitError` is hit because the task is not deleted upon completion.

Test Steps
-----------

Run demos on a board

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
